File: frameworks/rust/salvo/Implementation/routes/baseline.rs
use salvo::prelude::*;

// rb:handler baseline.plaintext
/// Salvo writes a `&'static str` as text/plain.
#[handler]
async fn plaintext() -> &'static str {
    "Hello, World!"
}

/// baseline: the dispatch floor, with nothing serialised.
pub fn router() -> Router {
    Router::with_path("/plaintext").get(plaintext)
}

File: frameworks/java/ktor/Implementation/src/main/kotlin/implementation/routes/Baseline.kt
package implementation.routes

import io.ktor.server.response.respondText
import io.ktor.server.routing.Route
import io.ktor.server.routing.get

/** baseline: the dispatch floor, with nothing serialised. respondText writes text/plain. */
fun Route.baseline() {
    get("/plaintext") { call.respondText("Hello, World!") }
}
